src/network/networkctl.c

```diff
@@ -1539,6 +1554,8 @@ static int link_status_one(
                         return table_log_add_error(r);
 
         } else if (streq_ptr(info->netdev_kind, "vxlan")) {
+                char str[MAX(strlen("auto") + 1, DECIMAL_STR_MAX(uint8_t))];
```

Comment:
gcc sucks and doesn't grok that strlen() is constant, still turns this into a VLA. please use STRLEN() here, i.e. the uppercase macro we defined which is guaranteed to be constant